### PR TITLE
Update Polish (pl) localization of Engagement 160726

### PR DESCRIPTION
**Pull request by piotrdrag**
needs-review: @splewako

### CODE REVIEW

File: pl/ads/ios_ads_nov2015.lang [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+
+# Text Ads (embedded in the image). Try to keep the same length as English where possible.
+;More than meets the iPhone.
+Coś więcej niż iPhone.
+
+
+# Text Ads (embedded in the image). Try to keep the same length as English where possible.
+;Firefox for iOS is here!
+Firefox na iOS jest już dostępny!
```
Comment (splewako) [inline pl/ads/ios_ads_nov2015.lang]: "Firefox na iOSa już jest!" would be shorter.

File: pl/snippets/2015/spring2015.lang [hunk 2]
```diff
@@ -5,151 +5,151 @@
...
-Get the official Firefox browser for Android and iOS devices. Email yourself a link to download it.
+Pobierz oficjalną przeglądarkę Firefox na Androida i iOSa. Wyślij sobie e-maila z odnośnikiem do pobrania.
 
 
 # LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
 # Campaign: iOS and Android
 # This is used in an interactive snippet with an input field and a button (see example at https://snippets.allizom.org/show/83/)
 ;Firefox for mobile? Yes, please! Get it for Android or iOS. Email yourself an instant download link.
-Firefox for mobile? Yes, please! Get it for Android or iOS. Email yourself an instant download link.
+Firefox na smartfony? Poproszę! Pobierz go na Androida i iOSa. Wyślij sobie e-maila z odnośnikiem do pobrania.
```
Comment (splewako) [inline pl/snippets/2015/spring2015.lang]: Please try not to use "smartfony", simple "telefony" would work good here.

File: pl/snippets/2015/spring2015.lang [hunk 3]
```diff
@@ -5,151 +5,151 @@
...
-Better mobile browsing is just a text message away. Free yourself with Firefox. Email yourself an instant download link.
+Lepsza przeglądarka za jednego SMS-a. Uwolnij się z Firefoksem. Wyślij sobie e-maila z odnośnikiem do pobrania.
 
 
 # LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
 # Campaign: iOS and Android
 # This is used in an interactive snippet with an input field and a button (see example at https://snippets.allizom.org/show/83/)
 ;Love Firefox on your computer? Add it to your mobile devices, too. Free yourself with Firefox. Email an official download link.
-Love Firefox on your computer? Add it to your mobile devices, too. Free yourself with Firefox. Email an official download link.
+Uwielbiasz Firefoksa na komputerze? Używaj go także na smartfonie. Uwolnij się z Firefoksem. Wyślij sobie e-maila z odnośnikiem do pobrania.
```
Comment (splewako) [inline pl/snippets/2015/spring2015.lang]: Please try not to use "smartfonie", simple "telefonie" would work good here.

File: pl/tiles/2015/tiles_jul2015.lang [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+## MAX_LENGTH: 45
+# LIMIT: average is about 32 characters. Make sure you check the demo site if using a longer string.
+;Try an add-on.
+Wypróbuj dodatek.
+
+
+## MAX_LENGTH: 45
+# LIMIT: average is about 32 characters. Make sure you check the demo site if using a longer string.
+;Sign up to learn more.
+Utwórz konto po więcej informacji.
```
Comment (splewako) [inline pl/tiles/2015/tiles_jul2015.lang]: "Utwórz konto, aby dowiedzieć się więcej"?

File: pl/tiles/2015/tiles_oct2015.lang [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+;Theme collection
+Kolekcja motywów
+
+
+# LIMIT: Try to keep about the same length of English. There are no automatic warnings for excessive length.
+# Title: Theme collection
+# State: normal (displayed without user interaction)
+# Link: https://addons.mozilla.org/firefox/collections/mozmj/eyes
+;The eyes have it
+Oczywiście oczy
```
Comment (splewako) [inline pl/tiles/2015/tiles_oct2015.lang]: Great one!

File: pl/ads/ios_ads_nov2015.lang (hunk 1 above)
Comment (piotrdrag) [inline pl/ads/ios_ads_nov2015.lang]: Fixed.

File: pl/tiles/2015/tiles_jul2015.lang (hunk 4 above)
Comment (piotrdrag) [inline pl/tiles/2015/tiles_jul2015.lang]: Fixed.

File: pl/snippets/2015/spring2015.lang (hunk 2 above)
Comment (piotrdrag) [inline pl/snippets/2015/spring2015.lang]: Fixed.

File: pl/snippets/2015/spring2015.lang (hunk 3 above)
Comment (piotrdrag) [inline pl/snippets/2015/spring2015.lang]: Fixed.

**Pull request merged**